src/App.js

```diff
@@ -1,23 +1,29 @@
-import logo from './logo.svg';
 import './App.css';
+import { Link, Route, Routes } from 'react-router-dom';
+import Home from './pages/Home';
+import AqiDetail from './components/AqiDetail';
+import PageNotFound from './pages/PageNotFound';
+import logoIcon from './assets/logoIcon.png';
+import Aqi from './pages/Aqi';
 
 function App() {
```

Comment:
- [ ] Kindly make sure that the home page of the app is following the design guidelines

|Your app|Provided design|
|-|-|
|![Screenshot_20230107_002401](https://user-images.githubusercontent.com/25227022/212152947-5c238804-5d1e-4ad5-b11d-0c448639a1aa.png)|![Screenshot_20230112_194223](https://user-images.githubusercontent.com/25227022/212152875-ae454f32-f4da-4f69-9133-257f1de69128.png)|